torch/csrc/jit/codegen/cuda/test/test_gpu_tensorcore.cpp

```diff
@@ -556,83 +721,441 @@ TEST_F(NVFuserTest, FusionAmpereMMATN_CUDA) {
   tv1cw->setMemoryType(MemoryType::Shared);
 
   auto options = at::TensorOptions().dtype(at::kHalf).device(at::kCUDA, 0);
-  auto t0 = at::randn({16, 16}, options);
-  auto t1 = at::randn({8, 16}, options);
+  auto t0 = at::randn({16, 16}, options);
+  auto t1 = at::randn({8, 16}, options);
+
+  FusionExecutor fe;
+  NVFUSER_TEST_CUDA_ARCH_COMPILE_CHECK(
+      8, 0, fe.compileFusion(&fusion, {t0, t1}));
+  auto cg_outputs = fe.runFusion({t0, t1});
+
+  auto tref = t0.to(at::kFloat).matmul(t1.t().to(at::kFloat));
+
+  testValidate(&fusion, cg_outputs, {t0, t1}, {tref}, __LINE__, __FILE__);
+}
+
+// MMA unit test on Ampere
+TEST_F(NVFuserTest, FusionAmpereMMATT_CUDA) {
```

Comment:
Is this test case using the circular or double buffering?

Reply:
Never mind, blaming how git is rendering the changes.

Reply:
Yes it's just rebasing noise. Will sort them out and cleanup all tests in follow up PRs.